astroplan/tests/test_core.py

```diff
@@ -786,6 +744,29 @@ def test_TargetNeverUpWarning(recwarn):
     assert issubclass(w.category, TargetNeverUpWarning)
     assert np.isnan(no_time)
 
+def test_timezone_convenience_methods():
+    location = EarthLocation(-74.0*u.deg, 40.7*u.deg, 0*u.m)
+    obs = Observer(location=location,timezone=pytz.timezone('US/Eastern'))
+    t = Time(57100.3, format='mjd')
+    assert (obs.astropy_time_to_datetime(t).hour == 3)
+
+    dt = datetime.datetime(2015, 3, 19, 3, 12)
+    assert (obs.datetime_to_astropy_time(dt).datetime ==
+            datetime.datetime(2015, 3, 19, 7, 12))
+
+    assert (obs.astropy_time_to_datetime(obs.datetime_to_astropy_time(dt)).replace(
+            tzinfo=None) == dt)
+
+    # Test vectors of times:
+    times = t + np.linspace(0, 24, 10)*u.hour
+    times_dt = times.datetime
+    assert all((obs.datetime_to_astropy_time(times_dt)).jd ==
```

Comment:
Very minor: you *might* want to also try it with ``list(times_dt)`` - in this test right now ``times_dt`` is an array of datetimes, but users may also pass in python lists. And as we've discussed, there are occasional subtle differences bewteen the two...